### DISCUSSION TITLE
Cask macupdater won't install - SHA256 mismatch

### DISCUSSION THREAD

**Discussion by backstab5983** [Homebrew/discussions#4749] [Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.7
ORIGIN: https://github.com/Homebrew/brew
HEAD: d4444b563e24ac7c05a93121c464c02dfa04d44f
Last commit: 3 days ago
Core tap JSON: 30 Aug 15:07 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 13.5.1-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Trying to install the cask macupdater, it seems impossible to due SHA256 mismatch, already tried to remove the file as brew recommends, run cleanup and try again but nothing. 

### Relevant casks

macupdater

**Comment by miccal** [comment, marked as answer]
https://github.com/Homebrew/homebrew-cask/pull/154086

**Comment by ErikAbele** [comment]
@miccal Looks like this happened again and maybe upstream changed the download file? At least I'm getting the following hash when downloading 3.3.0 of MacUpdater:

https://www.corecode.io/downloads/macupdater_3.3.0.dmg
b45971cf7d2fe93a6c49d15bee08f1587bbe154e31f2cef205e8111acfd96a5a

I did also comment on the original [commit](https://github.com/Homebrew/homebrew-cask/commit/e1a6f7bd94035959dca0b93c45addf65e891aa99#commitcomment-140867114).

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-cask/pull/171190
